packages/components/Checkbox/src/Checkbox.tsx

```diff
@@ -19,6 +21,14 @@ export const Checkbox = compose<CheckboxType>({
     required: Text,
   },
   useRender: (userProps: CheckboxProps, useSlots: UseSlots<CheckboxType>) => {
+    // Removal of tokens which are not supported for Android
+    {
+      Platform.OS === 'android' &&
+        unsupportedAndroidProps.map((e) => {
+          Reflect.deleteProperty(userProps, e);
+        });
+    }
```

Comment:
What is `Reflect`? Also, this differs from the approach Text took to filter out unsupported props on other platforms, which might have been re-usable

https://github.com/microsoft/fluentui-react-native/blob/037c4c54ac62d19633e03eb15d72084a568ab0f4/packages/components/text/src/Text.tsx#L109-L118

Reply:
Filtering of props would be useful only if they are being applied to component through code (like labelBefore/labelAfter would control where the Text component is being rendered), but if the prop is associated with a state, then there would be no advantage of removing it right? 
(Since the token files do not contain any specific tokens for the states associated with these props they will still not be applied anyway so it is already equivalent to props not being passed in the first place. ( Eg. Say size is not supported, then 'medium', 'large' would not have any token values specified for android, this would mean it would pick the default token values))

Reply:
Address on https://github.com/microsoft/fluentui-react-native/pull/2416 
As internal discussion , we will not remove props but don't show on tester app but still can be render on certain platforms.